x/twap/listeners.go

```diff
@@ -33,31 +35,35 @@ func (hook *epochhook) BeforeEpochStart(ctx sdk.Context, epochIdentifier string,
 	return nil
 }
 
-type gammhook struct {
+type swaprouterhook struct {
```

Comment:
I'm confused about the GammHook<>SwapRouterHook situation, what's the reason gamm hooks are called swaprouter hooks? (seeing that epochhooks are still epoch hooks, im generlly confused why gammhooks became swaprouter hook)